app/models/ability.rb

```diff
@@ -14,10 +14,14 @@ def initialize(user)
 
     unless user.new_record?
       can :create, Index
-      can :manage, Index, user_id: user.id
+      can :manage, Index, user_id: user.id, hidden: false
+    end
+
+    if user.index_admin? && namespace == 'manage'
+      can [:edit, :update], Index
```

Comment:
`:update` will catch the `:edit` action as well

probably need to see about splitting this up a bit to handle the different place we have to worry about permissions being checked from.

1) Normal index show, which won't be in the manage namespace - `can :hide, Index if user.index_admin?`
2) Manage index update, which will have the manage namespace - `can :update, Index`